### DISCUSSION TITLE
Can we live edit in front webpage and save back to source md file?

### DISCUSSION THREAD

**Discussion by shitou333111** [vuejs/vitepress#2158] [Q&A]
I want to edit content just on the production webpage (maybe fix some doc typo), and save back to the server source file.
Because i want anyone can do it, so auth doesn't matter, and i don't want to use github clone and commint process. I think from html content to md file is difficult, so i can accept the source file to be storaged as html (currently this is also not working). 
Can anyone tell me how to do it? Or is there any possible to do this? Thanks

**Comment by tassoman** [comment]
This software is supposed to compile static html, who need to be served, subsequentially, not the opposite.  
Maybe you better need a wiki software?
